wiki/Help_centre/Account/en.md

```diff
@@ -180,13 +181,13 @@ Your profile will show all your previous purchased name changes, but we only sel
 
 ### The flag/country on my account profile is wrong! {id=wrong-flag}
 
-**Unfortunately, we cannot change this for you.**
-
-The country associated with your osu! profile is based on the geographical location of the connection you used to register the account and is permanently assigned during account creation.
+**Yes, but only under certain circumstances.**
```

Comment:
this needs to be an answer to the heading, which contains a question different from the one you are answering

Reply:
oops didn't pay attention